presto-main/src/main/java/com/facebook/presto/execution/scheduler/SplitPlacementPolicy.java

```diff
@@ -26,4 +27,10 @@
     void lockDownNodes();
 
     List<Node> allNodes();
+
+    default Node anyNode()
```

Comment:
I'm not thrilled by this - why would a split placement policy have a method anyNode? It doesn't seem like the right fit. Might be worth pulling this back into SourcePartitionedScheduler.